validation/key-handover_test.go

```diff
@@ -0,0 +1,38 @@
+package validation
+
+import (
+	"testing"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func testKeyHandoverOfChain(t *testing.T, chainID uint64) {
+	rpcEndpoint := Superchains[OPChains[chainID].Superchain].Config.L1.PublicRPC
+	require.NotEmpty(t, rpcEndpoint, "no rpc specified")
+
+	client, err := ethclient.Dial(rpcEndpoint)
+	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
+
+	proxyAdmin, err := Addresses[chainID].AddressFor("ProxyAdmin")
+	require.NoError(t, err)
+
+	got, err := getAddress("owner()", proxyAdmin, client)
+	require.NoError(t, err)
+
+	want := Address(common.HexToAddress("0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A"))
```

Comment:
Should we do a check for the L2ProxyAdminOwner?

Reply:
Should this hardcoded address be declared in a TOML file somewhere?

Reply:
Can you add a comment here explaining what is special about that address? As in, why does the `ProxyAdmin.owner()` being set to this value indicate the keys have been handed over?